demisto_sdk/commands/format/update_layout.py

```diff
@@ -189,7 +189,11 @@ def set_layout_key(self):
             }
 
     def set_group_field(self):
-        if self.data["group"] != "incident" and self.data["group"] != "indicator":
+        if (
+            self.data["group"] != "incident"
+            and self.data["group"] != "indicator"
+            and self.data["group"] != "case"
```

Comment:
This pairing tends to repeat itself (incident + indicator + case) - usually when there's such a strong pairing it's better to unify them under a common constant, and use the constant for conditions.
Let's discuss this offline